jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditorDialog.kt

```diff
@@ -32,9 +33,10 @@ class QueryEditorDialog(
     private val queryingLogGroupApiCall = QueryingLogGroups(project)
     private val action: OkAction = QueryLogGroupOkAction()
     private val validator = QueryEditorValidator
+    private val logGroupNames = ArrayList<String>()
```

Comment:
in Kotlin, prefer `List` or `MutableList` over concrete implementations

Reply:
Changed it to Mutable List.